Improve tuning guide (#3155)
uyuni-project/uyuni-docs: modules/specialized-guides/pages/large-deployments/tuning.adoc

[cols="1,1"]
|===
| Description          | Before any action is executed on a client, a presence ping is executed to make sure the client is reachable.

>                          This parameter sets the amount of time before a second command (in most cases `state.apply` or any other salt function) is sent to the client to verify its presence.

Reviewer: ```suggestion
                         This parameter sets the amount of time before a second command (in most cases `state.apply` or any other {salt} function) is sent to the client to verify its presence.
```

[cols="1,1"]
|===
| Description          | Before any action is executed on a client, a presence ping is executed to make sure the client is reachable.

>                          After xref:java-salt-presence-ping-timeout[`java.salt_presence_ping_timeout`] seconds have elapsed without a response, a second command (in most cases `state.apply` or any other salt function) is sent to the client and if there is no response from the clinet for the amount of seconds specified with this parameter one more call (`saltutil.find_job`) is sent for a final check.

Reviewer: ```suggestion
                         After xref:java-salt-presence-ping-timeout[`java.salt_presence_ping_timeout`] seconds have elapsed without a response, a second command (in most cases `state.apply` or any other {salt} function) is sent to the client and if there is no response from the client for the amount of seconds specified with this parameter one more call (`saltutil.find_job`) is sent for a final check.
```

>                          This parameter sets the number of seconds after the second command after which the client is definitely considered timedout.

Reviewer: ```suggestion
                         This parameter sets the number of seconds after the second command after which the client is definitely considered timeout.
```

|===
| Description          | The number of `salt-master` worker threads that process commands and replies from minions and the Salt API.
                         Increasing this value, assuming sufficient resources are available, allows Salt to process more data in parallel from minions without timing out, but will consume significantly more RAM (typically about 70 MiB per thread).
                         Setting this value to very high values could cause opposite effect as the workers will compete to each other for the CPU resources and the performance could be dropped significantly.
| Tune when            | xref:client-count[Client count] increases significantly, xref:thread-pool[`thread_pool`] increases significantly, or `SaltReqTimeoutError` or `Message timed out` errors appear in `/var/log/salt/master` could be a sign of too low or too high value of this parameter.
| Value default        | 8

> | Value recommendation | 8-32, depending on the number of the CPU cores available for the server, it's recommended to keep the value slightly less than the number of CPU cores.

Reviewer: ```suggestion
| Value recommendation | 8-32, depending on the number of the CPU cores available for the server, it is recommended to keep the value slightly less than the number of CPU cores.
```
